pkg/dump/dump.go

```diff
@@ -40,6 +40,21 @@ func Pretty(stream io.Writer, body []byte) error {
 	return dumpJSON(stream, data)
 }
 
+// Simple functions exactly the same as Pretty except it uses jq's -c option to condense the 
+// output to a single line, intended to be used with other resources that require single line
+// output
```

Comment:
End the paragraph with a dot.

Reply:
@jhernand just to confirm, you mean:
```
// Simple functions exactly the same as Pretty except it uses jq's -c option to condense the 
// output to a single line, intended to be used with other resources that require single line
// output.
```
Right?

Reply:
Yes :-) .